pages/pipelines/command_step.md

```diff
@@ -306,6 +306,24 @@ _Optional Attributes_
       <em>Example:</em> <code>2</code>
     </td>
   </tr>
+  <tr>
+    <td><code>signal</code></td>
+    <td>
+      The signal causing the job to exit, if a signal was sent, and the currently running process did not intercept and handle the signal supplying a non-default exit status.<br>
```

Comment:
Okay, here's an attempt where I'm trying to strike a balance between everything we can tell them and what's helpful:

>The signal that causes the job to retry. The command process only receives a signal when it propagates from the agent and is not handled by another internal process like the bootstrap. We recommend using this for `SIGKILL` because it propagates reliably and differentiates between graceful cancelation and timing out.

What do you think? Does it make sense? The first sentence is the most important part. The rest is additional context that might help someone.

Reply:
I'd riffed on your words in the theme of front-loading more useful information. What do you think?